recipes/python-pathlib2/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "2.1.0" %}
+
+package:
+  name: python-pathlib2
+  version: {{ version }}
+
+source:
+  fn: pathlib2-{{ version }}.tar.gz
+  url: https://github.com/mcmtroffaes/pathlib2/archive/{{ version }}.tar.gz
+  md5: 5ef144f5050f17962e9cef35c0e93ed7
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - pathlib2
+
+about:
+  home: https://github.com/mcmtroffaes/pathlib2
+  license: MIT
+  summary: "Fork of pathlib aiming to support the full stdlib Python API."
+
+extra:
+  recipe-maintainers:
+    - pelson
+    - takluyver
```

Comment:
> I wonder if we could develop a more automated approach for straightforward packages, that wouldn't require human intervention on each new release. E.g. maybe convert wheels into conda packages?

There has been discussed many times with Continuum over a couple of years on this point of simplifying the entry of PyPI packages into the conda ecosystem. Recently, it sounded like there was some interest on there end. Though I don't know what timeline is being considered at present. Really it seems like that is where this change will need to begin for anything to happen on this front.

Even if it does happen, there are some packages that are just problems for this model of packaging. At least to me, it is not entirely clear how this gets automated on their end. How do they recognize whether a package can be converted this way or not? For instance, pyzmq is on PyPI, but I think we would rather build its C dependencies and compile pyzmq to link against them. Similarly, numpy is up there with ATLAS, but many conda users would be extremely unhappy using ATLAS. Not to mention, this falls into the same category as pyzmq. Namely, we would want to build the BLAS dependency separately. They can try to read the PyPI package metadata for C/C++/Cython and skip it, but that only goes so far. For example, Python libraries using `ctypes` or `cffi` still have C library dependencies that we should try to package if we can. At present, the best we can do is what we are doing now. Wrap up the Python dependencies we need here. This is already a significant improvement on the previous situation given the level of automation present here.

Within our own system, we have considered at the bare minimum trying to open [issues]( https://github.com/conda-forge/conda-forge.github.io/issues/51 ) on PyPI releases or when git tags are pushed. Ideally, these would be PR's with version updates. However, there is still a bit of diversity in our system that makes this not totally easy to automate. While desirable, there are simply more pressing problems. For awhile that has been CI efficiency and automation. Now, it is starting to seem like documentation and examples. Both before and now, a pressing issue has been community engagement, review of additions, and migration of conda-recipes. While this is going well thus far, there is still lots of work to do on that front. I think it is still some time off before we really sit down and evaluate this problem. That being said, we remain aware that it exists.

TL;DR - Sure, but it is not that simple. While it is an issue, there are other problems that are more pressing at present.